### PR TITLE
chore: updated documentation style

### PR DESCRIPTION
**Pull request by MattL75**
#### Please provide a link to the associated issue.
Part of #596 

#### Please provide a brief summary of this pull request.
Updated the styling for the documentation to be more like fd-react's.

I made some improvements to their design (at least in my opinion). First, elements aren't fixed-positioned. Instead we rely on a more robust flex layout. We also always show the collapse button, something which came recommended by designers. Otherwise, it's pretty much the same.

Docs are still a heavy WIP, but at least this makes it look nice

### CODE REVIEW

File: docs/styles.scss [hunk 1]
```diff
@@ -170,14 +168,6 @@ pre {
     padding: 10px 10px 0 10px;
 }
 
-.skip-navigation {
```
Comment (mikerodonnell89) [inline docs/styles.scss]: This is for screenreader users

**Comment by mikerodonnell89** [comment]
Seeing two scrollbars in the sidenav and neither of them work.  This is with Chrome
Screen Shot 2019-03-11 at 4 12 24 PM

**Comment by MattL75** [comment]
Hmm I can't see the two scrollbars with chrome, I'll see what I can do

**Comment by MattL75** [comment]
Issues fixed

**Comment by bcullman** [comment]
Consider hiding the "hamburger" menu until you hit the breakpoint to hide the left nav. If you leave it always visible like you do now, you will also need to track it's state. 

For example, on a large screen, hide the left nav by clicking the hamburger, then resize your window. The left nav returns because this is not stateful. 

Simpler to just set its visibility based on media queries alone.

**Comment by MattL75** [comment]
@bcullman I had that behaviour initially with the same reasoning, but our designers recommended keeping the hamburger. After hearing them out I see there's some sense in hiding the sidebar for some users that want to see only the doc text when they read, even on large screens.

**Comment by droshev** [comment]
🚢 looks good

**Pull request merged**